.ci/build-docker-images.groovy

```diff
@@ -36,24 +34,25 @@ pipeline {
   stages {
     stage('Checkout') {
       steps {
+        deleteDir()
         gitCheckout(basedir: "${BASE_DIR}",
           branch: "${params.BRANCH_REFERENCE}",
           repo: "https://github.com/elastic/${REPO}.git",
           credentialsId: "${JOB_GIT_CREDENTIALS}"
         )
         dir("${BASE_DIR}"){
-          setEnvVar("GO_VERSION", readFile(".go-version").trim())
+          setEnvVar("GO_VERSION", readFile(file: ".go-version")?.trim())
         }
       }
     }
-    stage('Install dependencies') {
-      when {
-        expression { return params.RELEASE_TEST_IMAGES }
-      }
-      steps {
-        sh(label: 'Install virtualenv', script: 'pip install --user virtualenv')
-      }
-    }
+    // stage('Install dependencies') {
```

Comment:
if the build ends correctly, it'd mean that is not need

Reply:
You are right, this shouldn't be needed now with python 3.